extensions/projection/examples/example-landsat8.json

```diff
@@ -0,0 +1,164 @@
+{
+  "id": "LC81530252014153LGN00",
```

Comment:
This example is incomplete/outdated. It is missing stac_version and stac_extensions, geometry, bbox and other item related fields?!

Reply:
updated